src/lib/index.jsx

```diff
@@ -8,7 +8,7 @@ export default function AddToCalendar(WrappedButton, WrappedDropdown) {
   return class AddToCalendarWrapped extends Component {
     static propTypes = {
       buttonProps: PropTypes.shape(),
-      buttonText: PropTypes.string,
+      buttonText: PropTypes.node,
```

Comment:
Since this no longer accepts _just_ text, maybe the prop name should change to reflect that. button _Text_ is no longer accurate.

Reply:
I am rendering other React components (an Icon) along with the "Add to Calendar" text:
```
buttonText={<Icon icon=["far", "calendar-alt"] /> Add to Calendar}
```

I was thinking about renaming too, but was concerned about potentially existing implementations.

I guess we can add a `buttonInner` prop and leave the buttonText alone?

Reply:
That makes sense. This doesn't seem like something we should force a breaking change with.